task2/app.py

```diff
@@ -3,32 +3,52 @@
 # 
 
 class PhoneBook:
+    
     def __init__(self) -> None:
-        pass
+        self.contacts = []
 
     def add(self, name: str, surname: str, phone: str) -> None:
-        """??? (https://peps.python.org/pep-0257/)"""
-        pass
+        """Adds a person to the phone book."""
+        for contact in self.contacts:
+            if contact['name'] == name and contact['surname'] == surname:
+                return
+        self.contacts.append({'name': name, 'surname': surname, 'phone': phone})
 
     def update(self, name: str, surname: str, phone: str) -> bool:
-        """??? (https://peps.python.org/pep-0257/)"""
-        pass
+        """Updates information about a person in the phone book."""
+        for contact in self.contacts:
+            if contact['name'] == name and contact['surname'] == surname:
+                contact['phone'] = phone
 
     def remove(self, name: str, surname: str) -> int:
-        """??? (https://peps.python.org/pep-0257/)"""
-        pass
+        """Removes a person from the phone book."""
+        for contact in self.contacts:
+            if contact['name'] == name and contact['surname'] == surname:
+                self.contacts.remove(contact)
+            return 1
```

Comment:
Should this `return` be under the `if`? 😄 

It's bad that all test were passed successfully. Please, fix this and add a new test to cover this behaviour